src/components/views/dialogs/LeaveSpaceDialog.tsx

```diff
@@ -41,7 +41,14 @@ const isOnlyAdmin = (room: Room): boolean => {
 };
 
 const LeaveSpaceDialog: React.FC<IProps> = ({ space, onFinished }) => {
-    const spaceChildren = useMemo(() => SpaceStore.instance.getChildren(space.roomId), [space.roomId]);
```

Comment:
Previously, getChildren gave us we had a list of all rooms and spaces within this space, I think.

Now, we get all rooms in this space, and then add all spaces in this space.  Is `getSpaceFilteredRoomIds` recursive into child spaces?  If not, I think we will miss any rooms that are in sub-spaces?

Reply:
`getSpaceFilteredRoomIds` is recursive, `getChildren` was not.
The traverse then sets the false flag so that it doesn't hit rooms, only spaces, to pick up the stragglers.